doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc

```diff
@@ -1,7 +1,6 @@
 [id='clustering-bc-configure-domain-proc']
 = Configuring a {EAP} 7.1 domain-mode cluster for {CENTRAL}
-
-By using domain mode with a cluster, you can centrally manage and publish configurations for your servers. Domain mode provides a central location to store and publish configurations. Complete the steps in this section to configure a {EAP} 7.1 cluster in domain mode, with Elasticsearch and the Artemis Java messaging server (JMS) broker resource adapter integrated with {EAP}. 
+You should configure the {CENTRAL} cluster in domain mode with Elasticsearch and the JMS broker resource adapter integrated with {EAP}. By using domain mode with a cluster, you can centrally manage and publish configurations for your servers.
```

Comment:
I am pretty sure "should" is not ok with the  style guide, for good reason - it is imprecise. "Should" means "probably do but perhaps not"? In what cases "not"? 

As we are documenting user stories, not all possible actions, how about a simple solution? "Configure the {CENTRAL} cluster ..."

Reply:
Ok :-)